js/everything.js

```diff
@@ -84,6 +85,33 @@ function onPlayerReady(event) {
     // Setup player
     if (currentVideoID) {
         ZenPlayer.init(currentVideoID);
+        setupPlyr();
+    }
+}
+
+function setupPlyr() {
+    //set up Plyr player
+    plyrPlayer = plyr.setup({
+        autoplay: true,
+        controls:["play", "current-time", "duration", "mute", "volume", "captions"]
+    })[0];
+    //Load video into Plyr player
+    if (plyrPlayer) {
+        plyrPlayer.source({
+            type: "video",
+            title: player.getVideoData().title,
+            sources: [{
+                src: currentVideoID,
+                type: "youtube"
+            }]
+        });
+        //Hide video; leave only controls
+        $(".plyr__video-wrapper").hide();
+        //Inject svg with controls' icons
```

Comment:
jQuery actually have a `load()` function, it's deprecated but I think it looks better than their hacky method

```javascript
$("#plyr-svg").load("../bower_components/plyr/dist/sprite.svg");
```